.pipelines/templates/template-acr-push.yml

```diff
@@ -0,0 +1,76 @@
+# This template is used to authenticate to Azure and push Docker images across Azure tenants.
+# We cannot use the simpler Docker@2 push command because MSI does not support cross-tenant authentication.
+
+parameters:
+  - name: jobName
+    type: string
+  - name: acrFQDN
+    type: string
+  - name: repository  # This is the repository in the ACR
+    type: string
+  - name: imageRepoName
+    type: string  # The local image repository name
```

Comment:
in all of the uses of these parameters, they are the same. Can we consolidate?

Reply:
- Consolidated `repository` and `imageRepoName` parameters into a single `repository` parameter.